index.php

```diff
@@ -16,9 +16,11 @@ class updater {
   public $server_url = 'https://updater.tangible.one';
   public $update_checkers = [];
   public $license_key_setting_field = 'license_key';
-
+  public $license_status_setting_field = 'status_key';
+  
   function load() {
     updater::$instance = $this;
+    $this->url = plugins_url( __FILE__, '/' );
```

Comment:
I wrote the wrong argument order in my comment, which I updated later. Based on the documentation for [plugins_url()](https://developer.wordpress.org/reference/functions/plugins_url/), it should be the other way around: the path to append and then the full path to the file. 

```suggestion
    $this->url = plugins_url( '/', __FILE__ );
```

I'm using the [Suggest Changes feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) of GitHub, so you can click to apply the change.